### PR TITLE
Add application.displayVersion and environment.settings.e10sCohort for "main" pings

### PR DESCRIPTION
**Pull request by georgf**
This adds the application.displayVersion property; leaving it optional
for now until we figure out versioning.

This also updates the validation sample to show-case the difference in
version numbers.

### CODE REVIEW

File: validation/telemetry/sample_v4_ping.json [hunk 1]
```diff
@@ -2,11 +2,12 @@
...
     "buildId": "20151103030248",
-    "channel": "nightly",
+    "channel": "beta",
     "name": "Firefox",
-    "platformVersion": "45.0a1",
+    "platformVersion": "45.0",
     "vendor": "Mozilla",
-    "version": "45.0a1",
+    "version": "45.0",
+    "version": "45.0b6",
```
Comment (mreid-moz) [inline validation/telemetry/sample_v4_ping.json]: s/version/displayVersion/

File: telemetry/main.schema.json [hunk 2]
```diff
@@ -33,6 +33,10 @@
           "type": "string",
           "pattern": "^\\d{2}\\."
         },
+        "displayVersion": {
+          "type": "string",
+          "pattern": "^\\d{2}\\."
```
Comment (mreid-moz) [inline telemetry/main.schema.json]: When will Firefox 99 come out? Do we need to worry about 3+ digits? Should we force the `version` field to "X.Y" where X and Y are positive ints?

Comment (sylvestre) [inline telemetry/main.schema.json]: yes, we should move to 3 digits to avoid potential issues, it is in a few years (about 6 if we keep the same rhythm).

Comment (georgf) [inline telemetry/main.schema.json]: We can bump it to 3 digits, but i wouldn't go restrict it further right now.
The current main goal seems to be to match the existing data, tightening the schema heavily is something we can look at later (i want this for client test validation etc. anyway).

**Comment by georgf** [comment]
@mreid-moz or @SamPenrose for review?

File: telemetry/main.schema.json [hunk 3]
```diff
@@ -27,11 +27,15 @@
         },
         "platformVersion": {
           "type": "string",
-          "pattern": "^[0-9]{2}\\."
+          "pattern": "^[0-9]{3}\\."
```
Comment (mreid-moz) [inline telemetry/main.schema.json]: Won't this now fail for 2-digit versions?  I am in favour of `"^[0-9]+\\."`, but if you want to limit to 2 or 3 digits only, we should use `"^[0-9]{2,3}\\."`

Comment (georgf) [inline telemetry/main.schema.json]: Good point, fixed and pushed.

**Comment by mreid-moz** [comment]
Looks good to me, r+

**Pull request merged**